docs/general/ledger.md

```diff
@@ -1,53 +1,61 @@
+:::warning Please Note
+
+- Because of required WebUSB support, Ledger wallets currently only work on Chromium-based browsers
+  like Google Chrome.
 
-Ledger does not currently support batch calls; As a consequence, if you stake using the
-[staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
-Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that allows
-for single calls (i.e., bonding and nominating).
+- Ledger does not currently support batch calls. As a consequence, if you nominate using the
+  [staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
+  Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that
+  allows for single calls (i.e., bonding and nominating). **Pool members are not affect by this
+  issue.**
 
 :::
 
-The Polkadot [Ledger](https://www.ledger.com/) application is compatible with the Ledger Nano S and
-the Ledger Nano X devices. Ledger devices are hardware wallets that keep your secret key secured on
-a physical device that does not expose it to your computer or the internet. Even if you connect your
-nano via USB to your computer, the private keys will not be leaked. Ledger devices are also
-hierarchical deterministic wallets (HD wallets), that is:
+The {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} [Ledger](https://www.ledger.com/)
+application is compatible with the Ledger Nano S and the Ledger Nano X devices. Ledger devices are
+hardware wallets that keep your secret key secured on a physical device that does not expose it to
+your computer or the internet. The private keys will not be leaked even if you connect your nano via
+USB to your computer. Ledger devices are also hierarchical deterministic wallets (HD wallets), that
+is:
 
 - _Deterministic_ means that only one seed phrase generates all the accounts for different
   blockchain networks.
 - _Hierarchical_ means that the accounts are generated in a tree-like structure for different
   purposes.
 
-Ledger devices can be equipped with applications that are blockchain-specific. Such applications are
-usually developed by third parties, and they enable the user to transact securely on the blockchain
-network. The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It
-supports most of the transaction types of the network, including batch transactions from the Utility
-pallet.
+Ledger devices can be equipped with applications that are blockchain-specific. Third parties usually
+develop such applications, enabling the user to transact securely on the blockchain network. The
+Polkadot Ledger application allows you to manage
+{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} native token,
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. It supports most of the transaction types of
+the network, including batch transactions from the Utility pallet.
```

Comment:
Here we are confirming that batch transactions from Utility pallet are supported.